index.html

```diff
@@ -351,6 +351,142 @@ <h4 class="skills-text">Skills</h4>
           </li>
         </ul>
       </article>
+
+      <section class="contact-me">
+        <h2 class="primary-header content">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get
+          in touch.
+        </h2>
+
+        <form action="https://formspree.io/f/mdobkdnq" method="POST" class="forms margin-centre">
+          <input
+            class="forms1"
+            type="text"
+            id="name"
+            name="name"
+            placeholder="Full name"
+            required
+            maxlength="30"
+          />
+          <input
+            class="forms1"
+            type="email"
+            id="email"
+            name="email"
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            class="forms1 forms2"
+            id="comment"
+            name="comment"
+            placeholder="Enter text here"
+            required
+            maxlength="500"
+          ></textarea>
+          <input
+            type="submit"
+            class="enable-button get-in-touch margin-centre"
+            value="Get in touch"
+          />
```

Comment:
Let's implement the button states.

![image](https://user-images.githubusercontent.com/36057474/147852588-9cabe7ab-8e68-4159-9edc-3b3024399d08.png)